hudi-integ-test/src/test/resources/unit-test-cow-dag.yaml

```diff
@@ -71,4 +71,24 @@ dag_content:
         query2: "select count(*) from testdb1.table1 group   by `_row_key` having count(*) > 1"
         result2: 0
     type: HiveQueryNode
-    deps: first_hive_sync
\ No newline at end of file
+    deps: first_hive_sync
+  first_presto_query:
+    config:
+      presto_props:
+        prop1: "SET SESSION hive.parquet_use_column_names = true"
+      presto_queries:
+        query1: "select count(*) from testdb1.table1"
+        result1: 300
+        query2: "select count(*) from testdb1.table1 group   by `_row_key` having count(*) > 1"
+        result2: 0
+    type: PrestoQueryNode
+    deps: first_hive_query
+  first_trino_query:
+    config:
+      trino_queries:
```

Comment:
so, trino does not require to set `SET SESSION hive.parquet_use_column_names = true` is it ?

Reply:
Not sure right now. In my local testing, it is not required. I'll check on EKS and if needed i'll add it in a followup PR.